### PR TITLE
Catch CannotGetPortalError of plone.volto:multilingual. Add testing profiles.

### PR DESCRIPTION
**Pull request by ksuess**
- Catch CannotGetPortalError of plone.volto:multilingual. 
Initializing content on robot server with profile plone.volto:multilingual fails by getting portal.
Why ever the portal is not grabbable in this context, catching and ignoring the CannotGetPortalError, is fine for testing.
So I just did this, catching, logging and skipping.  :-<

- Add testing profiles for single lingual and multi lingual site: english/german

### CODE REVIEW

**Comment by ksuess** [comment]
Hi @jensens 
I'm stuck here. This workarround is fine for testing. It's dirty, but I do not see a chance to solve it clean in an affordable amount of time. 
Can you please have a look. Please be pardoning.

**Comment by ksuess** [comment]
Ready for reviewing. ⛵

**Comment by ksuess** [comment]
merge and release please both collective.elastic.plone and collective.elastic.ingest

File: CHANGES.rst [hunk 1]
```diff
@@ -3,6 +3,7 @@ Changelog
 
 
 - Add @@clear-and-update-index-server-index. [ksuess]
+- Add profiles for testing single lingual and multi lingual. [ksuess]
```
Comment (jensens) [inline CHANGES.rst]: I would say "single lingual: is not the right term. I tend to "monolingual". But I asked for help here, my English is not good enough to decide on this.

**Comment by jensens** [comment]
Technical it is good and ready. Thanks. Just let wait until the comment above is solved.

File: CHANGES.rst (hunk 1 above)
Comment (jensens) [reply CHANGES.rst]: @ksuess Sorry for my pickiness, but there is no such term as `single lingual`. According to native speakers both, `single-language` or `monolingual`, are fine. Additional `monolingual`  and `multilingual` are written as one word.

Comment (ksuess) [reply CHANGES.rst]: If it's only this… That's easy to change. :-)

**Comment by ksuess** [comment]
@jensens Can we get a release please.

**Comment by jensens** [comment]
> @jensens Can we get a release please.

Done

**Pull request merged**